docs/administration/application-management.md

```diff
@@ -24,6 +24,14 @@ Dragging an application reference taken from the Eclipse Kura Marketplace to the
 
 ![](images/packageMarketplace.png)
 
+!!! warning
+    In some cases the installation from the Eclipse Marketplace fails for the lack of the correct certificates. In order to install the needed certificates, proceed as follows.
```

Comment:
This only works for add-ons coming from s3. Can we also provide instructions for older versions of Kura that have the old Eclipse certificate? Or maybe we can attach the new certificate? So that for the user is just a copy paste?